app/common/locales/en.json

```diff
@@ -57,6 +57,16 @@
             "post_types" : "Post Types",
             "all_types" : "All Types"
         },
+        "post_statuses" : {
+            "post_statuses" : "Post Statuses",
+            "all_statuses" : "All Statuses",
+            "draft" : "Draft",
+            "published" : "Published"
+        },
+        "post_stages" : {
+            "post_stages" : "Post Stages",
+            "all_types" : "All Types"
```

Comment:
Need to remove the extra "All types" string..
Maybe this could just be a single string rather than having `globalFilter.post_stages.post_stages` ?